src/auth/auth.module.ts

```diff
@@ -1,6 +1,4 @@
 import { NgModule, NgZone } from '@angular/core';
-import * as firebase from 'firebase/app';
-import 'firebase/auth';
```

Comment:
isn't it better using scoped one?
```js
import  '@firebase/auth';
// ...
```

Reply:
I am confused. This is simply a removal as it was not being used here. I did not add back any imports, only removed.

Reply:
Oh my bad. Sorry, I think I need some coffee.

Reply:
No worries, I feel ya. Been up since 7am yesterday (now 1pm). I am seeing cross eyed so I really was not sure if I was missing something!